app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,33 @@
-<h2>Log in</h2>
+</html> 
+<head>
+  <link href='https://fonts.googleapis.com/css?family=Graduate' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+</head>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
-  </div>
+<body>
```

Comment:
no need for `body` tag, this whole entire page is inserted within the `body` tag of `application.html.erb`. It replaces `<%= yeild %>`